daemon/src/battery/tplink.rs

```diff
@@ -0,0 +1,16 @@
+use std::path::Path;
+
+use crate::{
+    battery::{BatteryState, get_level_from_percentage_file, is_plugged_in_from_file},
+    error::RayhunterError,
+};
+
+const BATTERY_LEVEL_FILE: &str = "/sys/class/power_supply/bms/capacity";
+const PLUGGED_IN_STATE_FILE: &str = "/sys/devices/78d9000.usb/power_supply/usb/online";
```

Comment:
I think you might have misread [this comment](https://github.com/EFForg/rayhunter/issues/82#issuecomment-3111714841) which was actually about tmobile, not tplink.